src/main.rs

```diff
@@ -74,12 +74,14 @@ impl Process {
 }
 
 fn env_version(env_var_name: &str) -> Result<Option<String>> {
-    if let Some(raw_version) = env::var_os(env_var_name) {
+    let raw_version = env::var_os(env_var_name).unwrap_or(OsString::new());
+    if raw_version.len() == 0 {
+        // setting PANTS_VERSION= or PANTS_SHA= behaves the same as not setting them
```

Comment:
It would be great if this comment had a sibling test.

Reply:
Ah, I had grepped for `#[test]` and didn't find anything, but (embarrassingly) just concluded "oh, I guess there's no tests", rather than trying to grep for `test` alone. I've found them now, will add one!